src/signals/incident-management/containers/IncidentDetail/components/Detail/components/ExtraProperties/index.test.js

```diff
@@ -49,19 +48,36 @@ describe('<ExtraProperties />', () => {
     ]
   };
 
-  afterEach(() => {
-    jest.resetAllMocks();
-  });
+  afterEach(cleanup);
 
   it('should render correctly', () => {
-    wrapper = shallow(<ExtraProperties {...props} />);
+    const { queryAllByTestId } = render(
+      <ExtraProperties {...props} />
+    );
+
+    expect(queryAllByTestId('extra-properties-definition')).toHaveLength(6);
+    expect(queryAllByTestId('extra-properties-value')).toHaveLength(6);
 
-    expect(wrapper).toMatchSnapshot();
+    expect(queryAllByTestId('extra-properties-definition')[0]).toHaveTextContent(/^Gaat uw melding over één of over meer lampen\?$/);
```

Comment:
waarom deze opmerking? 
dat lijkt me juist wat je wil testen. helemaal als de tekst wijzigt. anders kan er ook broodje poep staan. ik bergrijp niet dat we hierover een gesprek aan het hebben.

Reply:
Because now you're testing a very specific implementation of a component's visual representation, not its functionality. How would this test be written if we're having to deal with translations?